roles/cdi/templates/cdi-operator.yaml.j2

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
```

Comment:
Delete the Namespace from here.  Let's cleanup the NS before we delete it.

Reply:
done